cmd/defaults.yaml

```diff
@@ -719,6 +719,9 @@ SystemDefaults:
       # If this is empty, the issuer is the requested domain
       # This is helpful in scenarios with multiple ZITADEL environments or virtual instances
       Issuer: "ZITADEL" # ZITADEL_SYSTEMDEFAULTS_MULTIFACTORS_OTP_ISSUER
+    RecoveryCodes:
+      # The number of recovery codes to generate for a user
+      Count: 5 # ZITADEL_SYSTEMDEFAULTS_MULTIFACTORS_RECOVERYCODES_COUNT
```

Comment:
Alternatively, should we just enforce the MaxCount or DefaultCount here and allow the "GenerateRecoveryCodes" request to specify the amount of codes to create?

Reply:
So whenever they used one, they could already create a new one, up to a max again of e.g. 5, right?
Even if the new list is only created once all are used, it would would allow more possibilities. So I think we could definitely go with the idea.

Reply:
Yeah that makes a lot of sense. I'll make that change